### PR TITLE
op-guide: add a guide for data migration

### PR DESCRIPTION
**Pull request by queenypingcap**
This document describes how to migrate data from MySQL to TiDB in detail.

### CODE REVIEW

**Comment by queenypingcap** [comment]
@c4pt0r @shenli @siddontang @ngaut  PTAL.

**Comment by c4pt0r** [comment]
LGTM
@shenli @siddontang

File: op-guide/migration.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+`syncer` outputs the current synchronized data statistics every 30 seconds:
+
+```bash
+2016/10/27 15:22:31 syncer.go:668: [info] [syncer]total events = 1, insert = 1, update = 0, delete = 0, total tps = 0, recent tps = 0, binlog name = mysql-bin.000003, binlog pos = 1280.
+2016/10/27 15:23:01 syncer.go:668: [info] [syncer]total events = 2, insert = 2, update = 0, delete = 0, total tps = 0, recent tps = 0, binlog name = mysql-bin.000003, binlog pos = 1538.
+```
+
+You can see that by using `syncer`, the updates in MySQL can be automatically synchronized in TiDB.
+
+
```
Comment (siddontang) [inline op-guide/migration.md]: remove the blank lines.

**Comment by siddontang** [comment]
Please add the link in README.md

**Comment by queenypingcap** [comment]
@siddontang @shenli PTAL.

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @shenli

**Comment by shenli** [comment]
LGTM

**Pull request merged**